src/Command/InfectionCommand.php

```diff
@@ -482,15 +482,18 @@ protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
 
-        $customConfigPath = $input->getOption('configuration');
-        $configExists = $customConfigPath && file_exists($customConfigPath);
+        if ($customConfigPath = $input->getOption('configuration')) {
+            $this->getContainer()->get('locator')->locate($customConfigPath);
```

Comment:
does it make sense to extract `$this->getContainer()->get('locator')` to variable? used twice